lib/rex/post/hwbridge/ui/console/command_dispatcher/rftransceiver.rb

```diff
@@ -108,8 +108,8 @@ def cmd_freq(*args)
     arg = {}
     opts = Rex::Parser::Arguments.new(
       '-h' => [ false, 'Help Banner' ],
-      '-f' => [ true, 'frequency to set, example: 433000000'],
-      '-m' => [ true, 'Mhz, default is 24']
```

Comment:
I removed the 'default is 24' here, as I couldn't see where 24 is being used as the default value, but I wanted to double-check that I'm not missing something.